lib/ui/window/window.cc

```diff
@@ -33,6 +35,24 @@ void Render(Dart_NativeArguments args) {
   UIDartState::Current()->window()->client()->Render(scene);
 }
 
+void SendPlatformMessage(Dart_Handle window,
+                         const std::string& name,
+                         Dart_Handle callback,
+                         const tonic::DartByteData& data) {
+  UIDartState* dart_state = UIDartState::Current();
+  blink::PlatformMessageSink sink = dart_state->platform_message_sink();
+  if (!sink)
+    return;
```

Comment:
Does it make sense to drop the callback on the floor? Maybe its better to assert here saying that the platform message sink has not been setup yet since that is a platform configuration error.